scripts/UpdateDependencies.ps1

```diff
@@ -81,13 +81,13 @@ if ($count -gt 0) {
     $ConfigVars += "--UpdatedVersions"
     $varString = ""
     foreach ($updatedVar in $updatedVars.GetEnumerator()) {
-        $varString += "$($updatedVar.Name)=$($updatedVar.Value);"
+        $varString += "$($updatedVar.Name)=$($updatedVar.Value)+"
```

Comment:
';' and ' ' don't work for the separator here, is there any danger in using '+'? If so what's a safer option?

Reply:
seems okay for now